plot_features.py

```diff
@@ -132,40 +78,66 @@ def plot_features_file(featurefile, mapInfo): #{{{
     # use colorbrewer qualitative 7 data class colors, "7-class Accent": http://colorbrewer2.org/
     colors = ['#7fc97f' ,'#beaed4', '#fdc086', '#ffff99','#386cb0','#f0027f','#bf5b17']
     markers = ['o', 's', 'v', '^', '>', '<', '*', 'p', 'D', 'h']
+    linestyle = ['-', '--', '-.', ':']
```

Comment:
Do we need different linestyles?  I had originally put this in but on further reflection it may be a bad idea because it could make seeing transects harder.